scripts/dockerimages/scripts/auto-format.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/bash
 
+set -e
+
+MAGIC="boot2docker, please format-me"
```

Comment:
Lets pass the MAGIC in as an env variable to the container so that we don't have to hard code it twice.